pkgs/by-name/on/onedrivegui/package.nix

```
@@ -58,7 +64,6 @@ python3Packages.buildPythonApplication rec {
   dontWrapQtApps = true;
 
   doCheck = false; # No tests defined
```

```suggestion
  doCheck = false; # No tests defined
  # pythonImportsCheck = [ "OneDriveGUI" ]; # requires a display
```

It may be possible to setup xvfb in installCheckPhase, but this is both brittle and not common.